src/sentry/models/transaction_threshold.py

```diff
@@ -15,6 +17,37 @@ class TransactionMetric(Enum):
     TransactionMetric.LCP.value: "lcp",
 }
 
+# This TTL is used to control how much the cached project threshold will be stored in cache. For now, because data
+# changes very infrequently, we opted for a TTL of 1 hour. The problem with such a long time is that it will create
+# cache coherence problems which in the worst case would happen if the model is changed right after the previous version
+# is inserted into the cache, in that case either we should invalidate the cache or we will have to deal with
+# out-of-date data for at most ~1 hour.
+PROJECT_TRANSACTION_THRESHOLD_CACHE_TTL = 3600
+
+
+def _get_cache_key(model_name, project_ids, org_id, order_by, value_list):
+    return "{}:{}".format(
+        model_name, md5_text(f"{project_ids}:{org_id}:{order_by}:{value_list}").hexdigest()
+    )
+
+
+def _filter_and_cache(cls, cache_key, project_id__in, organization_id, order_by, value_list):
```

Comment:
Please remove all naming to `project_id__in` everywhere to just `project_ids` except for when you are making this query

```suggestion
def _filter_and_cache(cls, cache_key, project_ids, organization_id, order_by, value_list):
```

Reply:
I agree! Thanks